hasjob/views/admin_filterset.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+from flask import flash, g
+from coaster.views import route, viewdata, UrlForView, ModelView
+from baseframe import __
+from baseframe.forms import render_form, render_redirect
+from .. import app, lastuser
+from ..models import db, Filterset
+from ..forms import FiltersetForm
+from .admin import AdminView
+
+
+@route('/f')
+class AdminFilterset(AdminView):
```

Comment:
AdminView is more appropriately named AdminTabbedView. This is not required if there are no tabs.